### PR TITLE
update kicktstarter a bit

### PR DESCRIPTION
**Pull request by dev7ch**
image

### CODE REVIEW

File: messages/de/app.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 
 return [
-    'kickstarter_success' => '<h1>Du hast es geschafft!</h1>
-    <p>Super, du hast es geschafft <i>LUYA</i> erfolgreich zu installieren. Du kannst dich nun in den Administrationbereich einloggen um dort deinen ersten Seiten zu erfassen.</p>
+    'kickstarter_success' => '<h1 class="alert-heading">Du bist bereit!</h1>
+    <p>Super, du hast es geschafft <i>LUYA</i> erfolgreich zu installieren. Du kannst dich nun in den Administrations-Bereich einloggen um dort deinen ersten Seiten erstellen.</p>
```
Comment (TheMaaarc) [inline messages/de/app.php:5]: "um dort deinen ersten Seiten erstellen" -> "und dort deine ersten Seiten erstellen"

File: messages/en/app.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'kickstarter_success' => '<h1>You are done!</h1>
+    'kickstarter_success' => '<h1 class="alert-heading">You are ready!</h1>
     <p>Yes, you have successful installed <i>LUYA</i> on your System. You are now able to login into the administration area to create your pages.</p>
```
Comment (TheMaaarc) [inline messages/en/app.php:5]: "successful" -> "successfully"
"login into" -> "log in to"

File: messages/en/app.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
     <p><strong>Important</strong><br />After loging into the administration area you have to navigation to &laquo;System&raquo; and enable all permissions for your group (Adminstrator).</p>',
```
Comment (TheMaaarc) [inline messages/en/app.php:6]: "loging into" -> "logging into"
"to navigation to" -> "to navigate to"

**Review by TheMaaarc — CHANGES_REQUESTED** [review]

File: messages/en/app.php (hunk 3 above)
Comment (dev7ch) [reply messages/en/app.php:6]: spelling and messages tide up will come in the next pr

Comment (dev7ch) [reply messages/en/app.php:6]: merge?

Comment (TheMaaarc) [reply messages/en/app.php:6]: Why not just do it right the first time?

Comment (dev7ch) [reply messages/en/app.php:6]: coz messages need top be reranged imo

Comment (dev7ch) [reply messages/en/app.php:6]: ... and this PR is already 'full and a bit too much changes'

Comment (TheMaaarc) [reply messages/en/app.php:6]: I'm not sure what you tried to say, but just remember my suggestions for when you fix those mistakes. 👍

**Review by TheMaaarc — APPROVED** [review]

File: messages/en/app.php (hunk 3 above)
Comment (dev7ch) [reply messages/en/app.php:6]: 🥇  i ll do

**Pull request merged**